jigna/tests/test_jigna_web.py

```diff
@@ -1,23 +1,49 @@
 from threading import Thread
+import time
+import sys
 import unittest
 from unittest import skip
-import time
 
 try:
     from tornado.ioloop import IOLoop
-    from selenium import webdriver
 except ImportError:
     raise unittest.SkipTest("Tornado not installed")
 
+try:
+    from selenium import webdriver
+except ImportError:
+    raise unittest.SkipTest("selenium not installed")
+
 # Local imports.
-from jigna.api import Template, VueTemplate, WebApp
 from jigna.utils.web import get_free_port
 from test_jigna_qt import TestJignaQt, Person, body_html
 
 
+def patch_sys_modules():
```

Comment:
Not sure if I'm missing something about how the PySide/PyQt modules are named, but could this be something along the lines of:-

`for name in ['PySide', 'PyQt']:
    if name in sys.modules.keys():
        del sys.modules[name]
`